js/serial.js

```diff
@@ -1,6 +1,9 @@
 'use strict';
 
 var serial = {
+    connectionType:  'serial', // 'serial' or 'tcp'
```

Comment:
after `failed`, before `transmitting`?
or after `outputBuffer`, before `connect`?
Sorry for my bad English.